specification/azsadmin/resource-manager/deployment/Microsoft.Deployment.Admin/preview/2019-01-01/ProductPackage.json

```diff
@@ -0,0 +1,270 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-01-01",
+    "title": "DeploymentAdminClient",
+    "description": "Deployment Admin Client."
+  },
+  "host": "adminmanagement.local.azurestack.external",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Deployment.Admin/locations/global/productPackages": {
+      "get": {
+        "x-ms-examples": {
+          "Return product packages list.": {
+            "$ref": "./examples/ProductPackage/List.json"
+          }
+        },
+        "description": "Returns an array of product packages.",
+        "tags": [
+          "ProductPackages"
+        ],
+        "operationId": "ProductPackages_List",
+        "parameters": [
+          {
+            "$ref": "Deployment.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Deployment.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProductPackagesList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Deployment.Admin/locations/global/productPackages/{productId}": {
+      "get": {
+        "x-ms-examples": {
+          "Return the product package details.": {
+            "$ref": "./examples/ProductPackage/Get.json"
+          }
+        },
+        "description": "Retrieves the specific product package details.",
+        "tags": [
+          "ProductPackages"
+        ],
+        "operationId": "ProductPackages_Get",
+        "parameters": [
+          {
+            "$ref": "Deployment.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Deployment.json#/parameters/ProductIdParameter"
+          },
+          {
+            "$ref": "Deployment.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProductPackage"
+            }
+          },
+          "404": {
+            "description": "Not Found"
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Creates a new product package.": {
```

Comment:
Is this the correct name of the example?

Reply:
It looks correct to me. Can you pls elaborate?

Reply:
Put it in the wrong place. Resolve this